README.md

```diff
@@ -192,4 +192,4 @@ SOFTWARE.
 [releases-shield]: https://img.shields.io/github/release/hassio-addons/bashio.svg
 [releases]: https://github.com/hassio-addons/bashio/releases
 [repository]: https://github.com/hassio-addons/repository
-[semver]: http://semver.org/spec/v2.0.0.htm
```

Comment:
Probably better as `https://semver.org/spec/v2.0.0`

This is also likely a challenge in all repos, but can be put into the next round of maintenance

Reply:
Adjusted. Thanks.